index.html

```diff
@@ -1,16 +1,114 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
```

Comment:
Stretch yourself to figure out what this exactly does, its pretty interesting How .html interfaces with stylesheets or js